resources/images/filters.go

```diff
@@ -87,6 +88,8 @@ func (*Filters) Text(text string, options ...any) gift.Filter {
 				tf.x = cast.ToInt(v)
 			case "y":
 				tf.y = cast.ToInt(v)
+			case "alignx":
+				tf.alignx = cast.ToString(v)
```

Comment:
Add some validation here. This method does not return error, so you need to panic, e.g. `panic("must be one of left, ...")`